### PR TITLE
add linux package install method

### PR DESCRIPTION
**Pull request by Krixa**
Linked to this [issue](https://github.com/paritytech/polkadot-sdk/issues/759).

### CODE REVIEW

**Comment by alfarok** [comment]
Thanks @Krixa, is this PR ready for review?

**Comment by Krixa** [comment]
> Thanks @Krixa, is this PR ready for review?

Yes !

File: docs/maintain/maintain-sync.md [hunk 1]
```diff
@@ -196,6 +197,87 @@ example:
...
+overwritten on updating polkadot.
+  
+### Debian-based (Debian, Ubuntu)
+
+Currently supports Debian 10 (Buster) and Ubuntu 20.04 (Focal), and
+derivatives. Run the following commands as the `root` user.
+
+```bash
+# Import the security@parity.io GPG key
+gpg --recv-keys --keyserver hkps://keys.mailvelope.com 9D4B2B6EB8F97156D19669A9FF0812D491B96798
```
Comment (alfarok) [inline docs/maintain/maintain-sync.md:220]: Hey @DrW3RK, this PR came in last week and I wanted to ask you about some of the technical details.  I am not very familiar with this setup process.  Is there someone else you would recommend for additional review?

**Comment by DrW3RK** [comment]
@Krixa Thank you very much for creating this PR to address https://github.com/paritytech/polkadot-sdk/issues/759

@bkchr Could you please review the content before we push this to the Wiki

**Comment by bkchr** [comment]
> @bkchr Could you please review the content before we push this to the Wiki

I'm probably the wrong one :D I didn't used Ubuntu for some years :P

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**